api/api.proto

```diff
@@ -558,6 +562,19 @@ message SdkAlertsEnumerateResponse {
     repeated Alert alerts = 1;
 }
 
+// SdkAlertsDeleteRequest is a request message to delete alerts.
+message SdkAlertsDeleteRequest {
+  // queries is a list of queries to find matching alerts.
+  // Output of each of these queries is added to a global pool
```

Comment:
This `Output of each of these queries is added to a global pool and returned as output of an RPC call.` is confusing. Does it mean, `Returns and deletes any alerts matching the query` ?